deploy/helm-chart/servicegateway/templates/ingress.yaml

```diff
@@ -1,14 +1,12 @@
 {{- if .Values.ingress.enabled -}}
 {{- $fullName := include "servicegateway.fullname" . -}}
+{{- $svcPort := .Values.service.port -}}
 apiVersion: {{ template "servicegateway.ingress.apiVersion" . }}
 kind: Ingress
 metadata:
   name: {{ $fullName }}
   labels:
-{{ include "servicegateway.labels" . | indent 4 }}
-{{- if .Values.additionalLabels }}
-{{ toYaml .Values.additionalLabels | indent 4 }}
-{{- end }}
```

Comment:
Why did you remove additional labels?
Without it, its not possible to add them.

Reply:
ups, good catch